src/Routing/RouteSubscriber.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * @file
+ * Apigee Edge UI module for Drupal.
+ *
+ * Copyright (C) 2019 PRONOVIX GROUP BVBA.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301,
+ * USA.
+ */
+
+namespace Drupal\apigee_edge_ui\Routing;
+
+use Drupal\Core\Routing\RouteSubscriberBase;
+use Symfony\Component\Routing\RouteCollection;
+/**
+ * Listens to the dynamic route events.
+ */
+class RouteSubscriber extends RouteSubscriberBase {
+  /**
+   * {@inheritdoc}
+   */
+  public function alterRoutes(RouteCollection $collection) {
+    // Replace "some.route.name" below with the actual route you want to override.
+    if ($route = $collection->get('entity.developer_app.collection_by_developer')) {
+      $route->setDefault('_controller', '\Drupal\apigee_edge_ui\BetterAppListBuilder::render');
```

Comment:
Just `Drupal\`.... Or even better,`BetterAppListBuilder::class . '::render'` because this way PHPstorm finds this as usage of `BetterAppListBuilder` class.

Reply:
@kittiradovics ^